src/sentry/integrations/utils/code_mapping.py

```diff
@@ -140,14 +140,36 @@ def list_file_matches(self, frame_filename: FrameFilename) -> list[dict[str, str
             ]
 
             for file in matches:
-                stacktrace_root, source_path = find_roots(frame_filename.raw_path, file)
+                stack_path = frame_filename.raw_path
+                source_path = file
+
+                try:
+                    stack_root, source_root = find_roots(stack_path, source_path)
+                except Exception:
```

Comment:
could we turn the [exception](https://github.com/getsentry/sentry/blob/dd5c494daa6d2ab070e255b9a67214e76b36a178/src/sentry/integrations/utils/code_mapping.py#L495) we're catching into a custom exception class? then we can catch just that custom exception instead of all exceptions (just in case this blanket catches exceptions we don't handle yet)